src/components/Authclient/CAuthclientEditorInfo.vue

```diff
@@ -81,7 +81,7 @@
           <b-form-input
             v-model="secret.value"
             disabled
-            placeholder="****************************************************************"
+            :type="secretType"
```

Comment:
Why are we changing this?

Reply:
Because previously secret was empty if we didn't previously explicitly push on the ("show") icon button to show the secret, only than was the API call to fetch the secret executed. 
Now we have another button "Generate cUrl snippet", which should be able to run weather user previously pushed on the reveal secret button or not.

Reply:
Not sure how all this is connected.

But I'm sure there must be a more elegant way to solve it.